Library/Formula/binwalk.rb

```diff
@@ -15,57 +13,61 @@ class Binwalk < Formula
   end
 
   head do
-    url 'https://github.com/devttys0/binwalk.git'
-    depends_on "automake" => :build
-    depends_on "autoconf" => :build
-
-    patch do
-      url "https://gist.github.com/balr0g/a917b31318016c63a72d/raw/d434c3ceaa57438d39344e760990e9268893ce5f/binwalk-head-201140712.patch"
-      sha1 "2e000654968d2c8ad17b7fc46924300cf01bbfed"
-    end
+    url "https://github.com/devttys0/binwalk.git"
   end
 
-  option 'with-matplotlib', 'Check for presence of matplotlib, which is required for entropy graphing support'
+  option "with-matplotlib", "Check for presence of matplotlib, which is required for entropy graphing support"
+  option "with-capstone", "Enable disasm options via capstone" if build.head?
```

Comment:
I might just say "(affects --HEAD only)" instead of adding `if build.head?` since it won't show up in `brew info` otherwise.